config/locales/views/mailers/user_mailer.fr.yml

```diff
@@ -17,3 +17,15 @@ fr:
         sign_me_in: Me connecter
         dont_hesitate_to_ask: N'hésitez pas à répondre à cet e-mail pour toute question !
         happy_discovery_html: Bonne découverte de %{link_to_root}.
+      yesterday_modifications:
+        subject: "[Réso] Hier sur Réso"
+        hello: Bonjour,
+        yesterday: 'Hier sur Réso les demandes suivantes ont évolué :'
+        context: "%{expert_name}, %{expert_institution}, contacté•e pour le besoin %{question_title} de l'entreprise %{company_name} le %{start_date},"
+        waiting_for_answer: n'a pas pris de decision sur le besoin.
+        taking_care: a pris en charge la résolution du besoin.
```

Comment:
`pris en charge le besoin` would suffice?

Reply:
I feel like the slight change in meaning is important